crates/sui-node/src/main.rs

```diff
@@ -32,6 +49,25 @@ async fn main() -> Result<()> {
         config.network_address = listen_address;
     }
 
+    use jemalloc_ctl::config;
+    let malloc_conf = config::malloc_conf::mib().unwrap();
+    info!("Default Jemalloc conf: {}", malloc_conf.read().unwrap());
+
+    std::thread::spawn(|| {
+        loop {
+            // many statistics are cached and only updated when the epoch is advanced.
+            epoch::advance().unwrap();
+
+            let allocated = stats::allocated::read().unwrap() / (1024 * 1024);
+            let resident = stats::resident::read().unwrap() / (1024 * 1024);
+            info!(
+                "Jemalloc: {} MB allocated / {} MB resident",
+                allocated, resident
+            );
+            std::thread::sleep(Duration::from_secs(60));
+        }
+    });
```

Comment:
Maybe we want to fact this into a common place seeing as its used in a number of different places?

Reply:
All of this code also needs to be appropriately marked with `#[cfg...]` so that its ignored in windows environments. We probably also only want to enable this when a flag is set, instead of unconditionally turning it on.

Reply:
I'll wrap stuff for non-windows.  However, the periodic stat update pretty much makes no impact to the runtime... it does make it easier to track the stat in the logs though, and we could track it as another memory stat too.